crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3132,6 +3132,7 @@ pub struct TypeParamTypeVar {
     pub range: TextRange,
     pub name: Identifier,
     pub bound: Option<Box<Expr>>,
+    pub default_value: Option<Box<Expr>>,
```

Comment:
My reasoning for this name was that it's the name being used in the CPython AST. I picked that name (well, Brandt Bucher suggested it) because `default` is a keyword in C, and AST field names get used as C identifiers in Python. I thought it'd be nice for consistency to reuse the name.

However, `default` isn't a keyword in Rust, so we could also use the shorter name here. Do you think consistency with the Python AST is useful here?

Reply:
I don't think it's required for the AST structure to be consistent with the Python AST as we've already diverged from it as per the needs of a static analysis tool. So, `default` should be fine here.

Reply:
Sure, moved to `default`.